install_files/securedrop-app-code/debian/rules

```diff
@@ -28,9 +28,11 @@ override_dh_virtualenv:
 	dh_virtualenv \
 		--python=/usr/bin/python3.5 \
 		--setuptools \
+		--builtin-venv \
```

Comment:
using --builtin-venv has the side-effect of adding `pyvenv.cfg` at the root of the `securedrop-app-code` virtualenv, which would require an update to the apache2 apparmor profile, if it is safe to include.

Reply:
> using --builtin-venv has the side-effect of adding `pyvenv.cfg` at the root of the `securedrop-app-code` virtualenv, which would require an update to the apache2 apparmor profile, if it is safe to include.

I have added the same.